blog.md

```diff
@@ -132,15 +132,15 @@ def chunks(l, n):
 batchedImages = chunks(image_list, 64)
 ```
 
-Now we have our images splited in batches of 64 we can upload them batch by batch to the Custom Vision Service. This can take a while!
+Now we have our images split in batches of 64, we can upload them batch by batch to the Azure Custom Vision Service. *Note: This can take a while!*
```

Comment:
can you get people to confirm the images uploaded by checking the UI? or does this take away from the code focus?